content/aro/shipping-logs-and-metrics-to-azure-blob/index.md

```diff
@@ -264,6 +264,22 @@ Next we can configure Metrics Federation to Azure Blob Storage. This is done by
    No resources found in mobb-aro-obs namespace.
    ```
 
+### Deploy OpenShift Path Operator
+
+1. Use Helm deploy the OpenShift Patch Operator
+
+    ```bash
+    helm upgrade -n patch-operator patch-operator --create-namespace \
+      mobb/operatorhub --version 0.1.2 --install \
```

Comment:
just tested this and it works but can you remove the version flag here please and thank you

Reply:
Good call out. That's been removed.